### PR TITLE
adding pypi configuration

### PR DESCRIPTION
**Pull request by brentru**
per https://github.com/adafruit/circuitpython/issues/979

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,34 +1,33 @@
...
   overwrite: true
   on:
     tags: true
-
+- provider: pypi
+  user: adafruit-travis
+  on:
+    tags: true
+  password:
+    secure: a2Qjgzums8nSfwPbwa5peExJgnajfhAZRkTqjFB1M35uTtw25yVfHPJAowiOgy4J2vwe8tzMsGHUKRSSFQN25oa26HD+HhB4TbGuP2TGbUJOzwVme3DH3XUPleanrKLJOkzmNp+p3Eu68IiLNsik2TUMXLWsV+sSOm52emvLLYv63siKZMSZzwhSqgQ0dWz4tWvKG+zcu6fOsFfbs3POAURQh4ULjQVKjtYK14aUv6XxPeB+FA1am6CJa7XQk64KJDWCcQHmKwtNcB/CznEXMSAogVneT2HncXpyJgwirjVUI4WP2X4xjfXCvi8QBq5Ya77YnvzIi/WGLhI8jkh/8HByb5fwVbm68cD1I33fQyHFphU5IA4RLxCEXSO43lXzYRARAXAJ59YcuP//QaUTBjsjt0mzavOcbZtvAHfK1k03NOoPsdjlYMkh+mXtciAysFrSzJ1XosGPTc9p99pGCJSs5ZMFSkJute0NB3ugQY/CAgxUCSEXS0uGc1PQdnFHtqAGMHjvCMIZx05iuHY+A3SZDx3WykX1cw8LKq/lhFYVIut2qaRXmefN8oLbJn7MjHiNbgSvZIDxZ2szv3xfTG/jeMnbgpTZzUNhgiJPnyz+NItPCF1gCnkOduSLKShUyL7D1N5szde41gx0r7oe5HO5vjN4x1qkqKDzvQY2DrM=
```
Comment (kattni) [inline .travis.yml:22]: It's odd that the hash split like this, I've not seen it do this before. Did you run the `travis encrypt` command that reformats the file itself, or did you run the version of the command that provides the hash and you manually add it to the file?

**Review by kattni — CHANGES_REQUESTED** [review]
Thanks for doing this! I have one question.

File: .travis.yml (hunk 1 above)
Comment (brentru) [reply .travis.yml:22]: I ran `travis encrypt --add deploy.password`

Comment (kattni) [reply .travis.yml:22]: Excellent! Should be good to go then. Thanks!

**Review by kattni — APPROVED** [review]

**Comment by kattni** [comment]
@brentru Please do a release with thorough release notes, and then verify Travis has passed and PyPI deployment has been successful after the release is complete. Thanks!

**Pull request merged**